collect_app/src/main/java/org/odk/collect/android/utilities/FileUtils.java

```diff
@@ -189,40 +189,24 @@ public static Bitmap getBitmapScaledToDisplay(File file, int screenHeight, int s
         BitmapFactory.Options options = new BitmapFactory.Options();
         options.inJustDecodeBounds = true;
         getBitmap(file.getAbsolutePath(), options);
```

Comment:
Since `if/else` block is removed and this instance of `options` is not being used to get the image's height/width, so this call can be removed. It is also important to understand that this call actually generates a new Bitmap from the given file path, so it does have an affect over performance and memory

Reply:
I agree that it generates a new Bitmap, but since it was already happening in the case when `upscaleEnabled=True` for the same test cases as when `upscaleEnabled=False`, I thought its effect on the performance was insignificant.